src/components/Input/Input.tsx

```diff
@@ -16,25 +16,17 @@ import { IInputProps } from './Input.types';
  * will render validation messages. If the user adds the required validator
  * then it will mark the field as required as well.
  */
-export const Input: React.FC<IInputProps> = (props) => {
+export const Input = <TSubmitValue extends unknown>(props: IInputProps<TSubmitValue>): JSX.Element => {
```

Comment:
This looks kinda terrible. I don't want to force users of ocean forms to write their form components like that!